app/views/participants/welcome.haml

```diff
@@ -7,7 +7,6 @@
   - if Participant.any?
     To view the scoreboard, click
     = link_to 'here', scoreboard_participants_path
-    or on the button above.
```

Comment:
naughty naughty

Reply:
If you're not signed in and you're on this page, there is no button! That was my only reasoning behind this, perhaps a little out of scope